openfisca_core/periods.py

```diff
@@ -673,6 +673,29 @@ def unit(self):
         return self[0]
 
 
+    # Reference periods
+
+    @property
+    def last_3_months(self):
+        return self.this_month.start.period('month', 3).offset(-3)
+
+    @property
+    def last_month(self):
+        return self.this_month.offset(-1)
+
+    @property
+    def n_2(self):
```

Comment:
Here are my 2 questions:

* `n_2` is not obvious for me without explanation since I didn't understand that the `_` had to be interpreted as `-`. We talked about naming it `2_years_ago` instead.

* Hardcoding the offsets in the function name isn't nice, what do you think about `last_months(2)` or `n_years_ago(2)`?

And/or some doc.